aws/container/sle15/root/lambda-entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+if [ $# -ne 1 ]; then
+    echo "entrypoint requires the handler name to be the first argument" 1>&2
+    exit 142
+fi
+
+if [ -z "${AWS_LAMBDA_RUNTIME_API}" ]; then
+    exec /usr/local/bin/aws-lambda-rie /usr/bin/python3 -m awslambdaric "$1"
```

Comment:
This will not exist. `/usr/bin/python3` points to `/usr/bin/python3.6` in SLE and openSUSE Leap. Since we know we are building only against Python 3.11 this should be `/usr/bin/python3.11`

Reply:
Actually I had planned to build the container such that it only includes python3.11 and nothing else... if possible. If not I agree

Reply:
My point is that the SLE python311 package does not set the `/usr/bin/python3` symlink. Basically this will fail unless some other operation sets up the `/usr/bin/python3` symlink pointing to `/usr/bin/python3.11`. If I remember correctly.

Reply:
And yes the container should only contain the Python 3.11 interpreter.

Reply:
done now. With the container only including the py3.11 stack there is no default choice for python3 in the system anymore. I added a config.sh script that explicitly selects it, which allows all other tools to reference `/usr/bin/python3`